src/types/runtime.go

```diff
@@ -55,6 +55,7 @@ type ZarfInspectOptions struct {
 type ZarfFindImagesOptions struct {
 	RepoHelmChartPath   string `json:"repoHelmChartPath" jsonschema:"description=Path to the helm chart directory"`
 	KubeVersionOverride string `json:"kubeVersionOverride" jsonschema:"description=Kubernetes version to use for the helm chart"`
+	RegistryOverride    string `json:"registryOverride" jsonschema:"description=Manual override for ###ZARF_REGISTRY###`
```

Comment:
```suggestion
	RegistryURL    string `json:"registryURL" jsonschema:"description=Manual override for ###ZARF_REGISTRY###`
```
Registry Override means something else in Zarf (--registry-override)